k256/src/arithmetic/scalar.rs

```diff
@@ -405,6 +542,12 @@ impl Generate for Scalar {
     }
 }
 
+impl Display for Scalar {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{:X}", self.to_bytes())
+    }
+}
```

Comment:
Wasn't sure exactly what a `Display` impl should do here, so I had it spit out upper case serialized hex